packages/wrangler/src/utils/ci-check.ts

```diff
@@ -0,0 +1,34 @@
+import ci from "ci-info";
+import { readConfig } from "../config";
+
+/**
+ * Inside a CI/CD environment, you want to check if all the required variables are in scope
+ * and if not, throw a helpful error message with missing variables and documentation link.
+ */
+export function ciCheck() {
+  if (ci.isCI) {
```

Comment:
thinking about this, why should we even bother checking whether it's in CI? if it's not a TTY, that should be enough?

Reply:
Done.